tests/queries/custom_aggregate/query.trickle

```diff
@@ -0,0 +1,22 @@
+define tumbling window by_two
+with
+  size = 2
+end;
+
+mod m with
+    aggregate collect_joined of
+        init ( ) =>
+            "not an empty string"
+
+        aggregate(st, x) =>
+            st + x
+
+        mergein(st, x) =>
```

Comment:
We probably want to look at a better name then `mergin`

Reply:
I agree :). I have a hard time coming up with anything though, any suggestions? `merge` is causing conflicts in the grammar, since it's a keyword

Reply:
Brainstorming (so those are not necessarily good ideas :joy:):

* Moving it first could help with the grammar, adding `end` to each section could also help with the grammar (but might not look nice and would break the existing pattern of case)



alternative names:
* `combine` - just another word for merge
* `tilt` - we call the concept of chained windows `tilt frames` so calling the function that ingests the previous window into this `tilt` but it will require more domain knowledge and understanding that `combine`
* `into` - that is already a keyword so might be a win of not adding a newone and the function takes state from another window and puts it `into` this one 
* `consume` or `ingest` - to represent that we're taking state in
* `append` since we're adding the state of another window to this
* `add` - it adds two aggregates together (possibly misleading with adding a singular value, but it's what hdr histogram uses)